docs/2.8.0/docs/daml/reference/templates.rst

```diff
@@ -92,7 +92,52 @@ Template-local Definitions
    - interface methods applied to ``this``
 
    in a template-local let definition will result in an error generated by the infinite loop.
- 
+
+.. warning::
+   Since Daml 2.8.0, template-local definitions are deprecated and their
```

Comment:
I added this in the docs for 2.8.0 given that 2.7.0 is about to be released. Perhaps 2.7.1 would make more sense but we don't have a folder for that yet.

Reply:
2.7.0 is not out yet, if this is something you and the team feel strongly about there is still time to put it in the release notes and add that one line to the docs.

Reply:
(We definitely don't deprecate things in patch releases, so it's either 2.7.0 or 2.8.0.)